packages/daimo-common/src/foreignCoin.ts

```diff
@@ -66,3 +66,15 @@ export function isAmountDust(
 
   return true;
 }
+
+// Get home coin token for a given token address.
```

Comment:
not sure we need this function

what do you mean by "home coin token for a given token address"?

home coin is a property of a daimo account

the function behavior below is equivalent to just returning `daimoUSDC` in all cases, but with the address potentially overwritten to a different address? seems like a footgun